apps/condo/domains/resident/tasks/helpers/sendBillingReceiptsAddedNotifications.js

```diff
@@ -101,7 +101,7 @@ const prepareAndSendNotification = async (context, receipt, resident) => {
 const sendBillingReceiptsAddedNotificationsForPeriod = async (receiptsWhere, onLastDtChange ) => {
     const { keystone: context } = await getSchemaCtx('Resident')
     const receiptsCount = await BillingReceipt.count(context, receiptsWhere)
-    let skip = 0, successCount = 0
+    let skip = 0, successCount = 0, notifiedUsers = {}
```

Comment:
Why don't use a `Set` (`add()` and `has()`)? It is faster and utilizes less memory.
```suggestion
    let skip = 0, successCount = 0
    const notifiedUsers = new Set()
```

Reply:
fixed